### PR TITLE
Update PredictionIO to latest version (0.9.5)

### PR DESCRIPTION
**Pull request by schonfeld**
This updates the download URL & checksum for PredictionIO's latest version - 0.9.5.

### CODE REVIEW

File: Library/Formula/predictionio.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Predictionio < Formula
   desc "Source machine learning server"
   homepage "https://prediction.io/"
-  url "http://download.prediction.io/PredictionIO-0.9.2.tar.gz"
-  sha256 "e4da196a1d67f545d6667f8368a7c6a64ff5c6de5135b085a12e9251b6076991"
+  url "https://d8k1yxp8elc6b.cloudfront.net/PredictionIO-0.9.5.tar.gz"
```
Comment (DomT4) [inline Library/Formula/predictionio.rb]: I'd be tempted to keep this at the `http://download.prediction.io` download URL. Random looking Cloudfront URLs can tend to feel a bit shady, even when that isn't the case.

Comment (schonfeld) [inline Library/Formula/predictionio.rb]: Fair point. Changing now...

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/b4b28a5419ebfa6ad0e2204d8eca497173249d1a. Thank you for your contribution to Homebrew @schonfeld; we appreciate it! :smiley_cat:

**Comment by schonfeld** [comment]
Woot! Thanks, @DomT4.

**Pull request closed without merging**